packages/app/src/containers/Canvas.tsx

```diff
@@ -734,26 +735,57 @@ export default memo(function Canvas() {
     }
   }, [state, handleDirection]);
 
+  const onDropFile = useCallback(
+    async (file: File, extension: string, e: MouseEvent) => {
```

Comment:
Since we only use the event for getting the drop location, we could pass just a point instead of the event object, and update our `getPoint` function to use the same type.

```ts
export type OffsetPoint = {
  offsetX: number;
  offsetY: number;
};

function getPoint(event: OffsetPoint): Point { ... }
```